src/Fixture/ShopUserFixture.php

```diff
@@ -4,8 +4,23 @@
 
 namespace Synolia\SyliusMailTesterPlugin\Fixture;
 
+use Doctrine\Persistence\ObjectManager;
+use Sylius\Bundle\CoreBundle\Fixture\Factory\ExampleFactoryInterface;
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+use Synolia\SyliusMailTesterPlugin\Fixture\Factory\ShopUserExampleFactory;
+
+#[AutoconfigureTag('sylius_fixtures.fixture')]
 final class ShopUserFixture extends \Sylius\Bundle\CoreBundle\Fixture\ShopUserFixture
```

Comment:
Weird, I don't remember the purpose of this fixture. Maybe we should deprecate it ?